Sources/SwiftDocC/Infrastructure/Link Resolution/LinkResolver+NavigatorIndex.swift

```diff
@@ -116,7 +117,7 @@ struct NavigatorExternalRenderNode: NavigatorIndexableRenderNodeRepresentation {
             navigatorTitle: renderNode.navigatorTitleVariants.value(for: traits),
             externalID: renderNode.externalIdentifier.identifier,
             role: renderNode.role,
-            symbolKind: renderNode.symbolKind?.identifier,
+            symbolKind: renderNode.symbolKind?.renderingIdentifier,
```

Comment:
From https://github.com/swiftlang/swift-docc/blob/1b4a1850dd2785a8ebabded139ae0af3551bb029/Sources/SwiftDocC/Semantics/Graph/SymbolKind.Swift.swift#L27 it seems like ideally we don't want to use the renderingIdentifier.